components/tutorial/tutorial_tip.jsx

```diff
@@ -16,31 +16,28 @@ import tutorialGif from 'images/tutorialTip.gif';
 import tutorialGifWhite from 'images/tutorialTipWhite.gif';
 
 const Preferences = Constants.Preferences;
+const TutorialSteps = Constants.TutorialSteps;
 
 export default class TutorialTip extends React.Component {
     constructor(props) {
         super(props);
 
         this.handleNext = this.handleNext.bind(this);
-        this.toggle = this.toggle.bind(this);
+        this.show = this.show.bind(this);
+        this.hide = this.hide.bind(this);
         this.skipTutorial = this.skipTutorial.bind(this);
 
         this.state = {currentScreen: 0, show: false};
     }
-    toggle() {
-        const show = !this.state.show;
-        this.setState({show});
-
-        if (!show && this.state.currentScreen >= this.props.screens.length - 1) {
-            const step = PreferenceStore.getInt(Preferences.TUTORIAL_STEP, UserStore.getCurrentId(), 0);
-
-            savePreference(
-                Preferences.TUTORIAL_STEP,
-                UserStore.getCurrentId(),
-                (step + 1).toString()
-            );
-        }
+
+    show() {
```

Comment:
I've eliminated `toggle` in favour of explicit `show` and `hide` from the requisite places.